packages/v1-sdk/src/actions/lp/simulateEditLp.ts

```diff
@@ -11,6 +6,7 @@ import { getPeripheryContract } from '../../common/contract-generators';
 import { getMarginEngineContract } from '../../common/contract-generators';
 import { PositionInfo } from '../../common/api/position/types';
 import { getPositionInfo } from '../../common/api/position/getPositionInfo';
+import { decodeV1PositionId } from '@voltz-protocol/commons-v2/dist/types';
```

Comment:
hmm, can we import the ts file instead of from dist?

Reply:
nice spot. It's .vscode auto-completed. I've fixed it.